src/iologindata.cpp

```diff
@@ -470,6 +472,54 @@ bool IOLoginData::loadPlayer(Player* player, DBResult_ptr result)
 		}
 	}
 
+	//load reward chest items
+	itemMap.clear();
+
+	query.str(std::string());
+	query << "SELECT `pid`, `sid`, `itemtype`, `count`, `attributes` FROM `player_rewards` WHERE `player_id` = " << player->getGUID() << " ORDER BY `sid` DESC";
+	if ((result = db->storeQuery(query.str()))) {
+		loadItems(itemMap, result);
+
+		//first loop handles the reward containers to retrieve its date attribute
+		//for (ItemMap::iterator it = itemMap.begin(), end = itemMap.end(); it != end; ++it) {
+		for (auto& it : itemMap) {
+			const std::pair<Item*, int32_t>& pair = it.second;
+			Item* item = pair.first;
+
+			int32_t pid = pair.second; 
```

Comment:
Perhaps it should, but I'd rather sacrifice clarity in this case, instead of having: A) to change every other ocurrence of ````pid```` both in the code and in the database; B) ````player_rewards```` use ````parent_id```` and ````player_depotitems, player_inboxitems and player_items```` use ````pid````.